README.md

```diff
@@ -417,7 +417,13 @@ Commit the result.
     docker tag cpmonitor:latest cpmonitor:${DATESTR}
     docker tag klimaschutzmonitor-dbeaver:latest klimaschutzmonitor-dbeaver:${DATESTR}
     ```
-9. Stop the server, apply the migrations, start the server:
+9. Stop the reverse proxy:
+    ```sh
+    cd docker/reverseproxy
+    docker-compose down
```

Comment:
Is it really necessary to close access to both test and production systems, when we do update to just one of them? Would it maybe be possible to put the network config into the reverse proxy config? Or does that have other disadvantages?

Reply:
I think in principle you can also leave the reverse proxy running. Docker will inform you that it could not close the testing/production network when you stop the testing/production containers. And I think (but one should test it again) that it connects to the correct network when you restart the testing/production container.

Reply:
Yes, that's how it works - compose complains that it couldn't remove the network and keeps it instead, and the proxy can keep running and will reach the app again once the app has restarted.
I'll remove this step from the README.